packages/table/examples/tableEditableExample.tsx

```diff
@@ -47,6 +47,7 @@ export class TableEditableExample extends BaseExample<{}> {
         return (<EditableCell
             value={value == null ? "" : value}
             intent={this.state.sparseCellIntent[dataKey]}
+            loading={true}
```

Comment:
oops, i was testing and forgot to change this back. there isn't an example for editable cell loading. the behavior is the same as Cell, so i was thinking we could just mention it in the docs